src/main.cpp

```diff
@@ -6,11 +6,13 @@
 #include "pico/cyw43_arch.h"
 
 #include "lwipopts.h"
+#include "cgi.h"
 #include "ssi.h"
 
 void run_server() {
     httpd_init();
     ssi_init();
+	cgi_init();
```

Comment:
nit-pick: please fix the indent here
```suggestion
    cgi_init();
```